README.md

```diff
@@ -351,12 +338,7 @@ And worrying about someone else's _"experience"_ with your app can be a full-tim
 However, it is an essential process for everyone involved in software to partake in.
 The very best pieces of software are differentiated by the quality of their **User Experience design**.
 
- Experiences are, by nature, personal, contextual, ephemeral, and affected by things beyond your control.
- The job of designing one is essentially one of embodying the _real people_ that will be using your product
- and imagining what the experience would be like for them.
- You should try to take into account as much contextual information about your users state of mind, occupation, preference towards sans-serif fonts etc.
- These details will weave a picture of the real humans on the other end of your app
- and help you make decisions about site structure and navigation as well as other less obvious things like content hierarchy: _what should the user see first?_
+Experiences are, by nature, personal, contextual, ephemeral, and affected by things beyond your control. The job of designing one is essentially one of embodying the _real people_ that will be using your product and imagining what the experience would be like for them. You should try to take into account as much contextual information about your users state of mind, occupation, preference towards sans-serif fonts, etc. These details will weave a picture of the real humans on the other end of your app and help you make decisions about site structure and navigation as well as other less obvious things like content hierarchy: _what should the user see first?_
```

Comment:
Same here, we _prefer_ to break paragraphs into semantic line breaks. https://sembr.org